spec/fuzzy-finder-spec.coffee

```diff
@@ -1075,3 +1075,15 @@ describe 'FuzzyFinder', ->
 
           runs ->
             expect(projectView.list.find("li:contains(file.txt)")).toExist()
+
+      describe "maxItem config limitation", ->
```

Comment:
It's more important that the configuration is reflected in the view class than we do bounds checking in the specs. I'd like to see the following specs:

1. Instantiate the fuzzy finder view without changing the configuration and verify that the default is 10 as expected
1. Change the configuration (within the min and max values), instantiate a view and verify that the view has the right value
1. (Optional) Set the value to something out of bounds, instantiate a view and verify that the view has an in-bounds value